app/src/main/java/org/squidfish/tuplenback/games/engines/SoundGameEngine.kt

```diff
@@ -8,7 +8,9 @@ import android.util.Log
 class SoundGameEngine(recallsBack: Int, repeatChance: Int) : GameEngine(recallsBack, repeatChance) {
     override val gameButtonText = "Sound"
 
-    override fun genNewMnemonic(forbidden: List<Int?>): Int = keys.indices.filterNot { forbidden.contains(it) }.random()
+    override fun genNewMnemonic(forbidden: List<Int>): Int = keys.indices.filterNot { forbidden.contains(it) }.random()
+
+    override fun onNewRound(mnemonicId: Int) {}
 
     companion object {
         val keys = Array(24) { i ->
```

Comment:
Could a list of resource ids be passed as a constructor parameter to `SoundGameEngine` instead?